gripper/core/tasks.cpp

```diff
@@ -85,11 +85,11 @@ void gripper_tasks::start_tasks(
 
     auto& capacitive_sensor_task_front =
         capacitive_sensor_task_builder_front.start(
-            5, "capacitive sensor S0", i2c2_task_client, i2c2_poll_client,
+            5, "capacitive sensor S1", i2c2_task_client, i2c2_poll_client,
```

Comment:
Not related to this PR, but we should change the name of these tasks to "cap sensor Sn" since our task name length is only 16 chars